iphone/Classes/TiUITextWidget.m

```diff
@@ -160,6 +160,70 @@ -(void)setAutocorrect_:(id)value
 	[[self textWidgetView] setAutocorrectionType:[TiUtils boolValue:value] ? UITextAutocorrectionTypeYes : UITextAutocorrectionTypeNo];
 }
 
+-(void)setTextContentType_:(id)value
+{
+    ENSURE_TYPE_OR_NIL(value, NSString);
+    if (@available(iOS 10.0, *)) {
```

Comment:
if (![TiUtils isiOS10OrGreater]) {
        NSLog(@"Setting a text-content-type is only available on iOS 10 and later");
        return;
    }  , I agree. 

  With  [[self textWidgetView] setTextContentType:[TiUtils stringValue:value]], I have already tried but it didn't worked. When I saw the class type of [TiUtils stringValue:value], it gives value	"NSTaggedPointerString " and for content type "__NSCFConstantString". This api accept only Constant string.
So we have to make the method large with manual logic.